tests/SchemaTest.php

```diff
@@ -30,6 +30,18 @@ public function testGetIndexSchemas()
         }
     }
 
+    public function testGetPrimaryKeySchema() {
+        $indexes = $this->getConnection()->schema->getIndexSchemas();
+        foreach($indexes as $index) {
+            foreach($index->columns as $column) {
+                if ($column->isPrimaryKey) {
+                    $this->assertEquals($index->primaryKey, $column->name);
```

Comment:
Extra test never will be superfluous.
Howevere, does this test expose the reported issue? It seems it does not.